app/pages/project/about/index.cjsx

```diff
@@ -0,0 +1,69 @@
+React = require 'react'
+TitleMixin = require '../../../lib/title-mixin'
+{Markdown} = require 'markdownz'
+ChangeListener = require '../../../components/change-listener'
+PromiseRenderer = require '../../../components/promise-renderer'
+Translate = require 'react-translate-component'
+counterpart = require 'counterpart'
+{Link} = require 'react-router'
+
+counterpart.registerTranslations 'en',
+    nav:
+        research: 'Research'
+        results: 'Results'
+        faq: 'FAQ'
+        education: 'Education'
+        team: 'The Team'
+
+module.exports = React.createClass
+  displayName: 'About'
+
+  mixins: [TitleMixin]
+
+  title: 'About'
+
+  getDefaultProps: ->
+    project: null
+
+  getPageTitles: (page) ->
+    page.filter((page) -> page.content isnt '' and page.content?)
+      .reduce(((accum, page) -> accum[page.url_key] = page.title; accum), {})
+
+  render: ->
+    <ChangeListener target={@props.project}>{=>
+      <PromiseRenderer promise={@props.project.get 'owner'}>{(owner) =>
+        [ownerName, name] = @props.project.slug.split('/')
+        projectPath = "/projects/#{ownerName}/#{name}"
+
+        <div className="about-container">
+          <PromiseRenderer promise={@props.project.get 'pages'}>{(pages) =>
+            pageTitles = @getPageTitles(pages)
+            <span>
+              {if pageTitles.science_case
+                <Link to="#{projectPath}/about/research" activeClassName="active"className="about-tabs">
+                  <Translate content= "nav.research" />
+                </Link>}
+              {unless @props.project.redirect
```

Comment:
The default team page (if not modified in the lab) will display a list of all collaborators, etc. and their Zooniverse ID. If a team page contains content, the content will be displayed with the Zooniverse IDs shifted to the right. Because of this, I'm thinking it's best to always make the Team page tab/link available.